browser/brave_wallet/brave_wallet_service_unittest.cc

```diff
@@ -3267,6 +3267,139 @@ TEST_F(BraveWalletServiceUnitTest, MaybeMigrateCompressedNfts) {
   EXPECT_TRUE(GetPrefs()->GetBoolean(kBraveWalletIsCompressedNftMigrated));
 }
 
+TEST_F(BraveWalletServiceUnitTest, MaybeMigrateSPLNfts) {
+  GetPrefs()->SetBoolean(kBraveWalletIsSPLTokenProgramMigrated, false);
```

Comment:
Should be EXPECT_FALSE instead of setting it here.

Reply:
In this case we need to set it explicitly since the migration is already run once before the test is run